src/Symfony/Component/Lock/StoreInterface.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Symfony\Component\Lock;
 
+@trigger_error(sprintf('"%s" is deprecated since Symfony 4.4 and has been splitted into "%s", "%s", "%s".', StoreInterface::class, PersistStoreInterface::class, ExpiringStoreInterface::class, BlockingStoreInterface::class), E_USER_DEPRECATED);
```

Comment:
you need to remove this deprecation warning, as this will get triggered when the interface is loaded, even though it is loaded by our BC layer. Details about the deprecation and the replacement should be in the `@deprecated` annotation, as DebugClassLoader will use it to generate the deprecation warning when implementing the deprecated interface in a separate package.

On a side note, there is a typo in your message. `to split` is an irregular verb: https://www.englishforums.com/English/SplittedIsItCorrect/bbxlql/post.htm